server/app/featureflags/FeatureFlags.java

```diff
@@ -62,7 +56,7 @@ public boolean getFlagEnabled(Request request, FeatureFlag flag) {
     }
     Boolean configValue = maybeConfigValue.get();
 
-    if (!areOverridesEnabled()) {
+    if (!overridesEnabled() || request == null) {
```

Comment:
**suggestion (blocking):** please use `Optional` rather than null: https://docs.civiform.us/contributor-guide/developer-guide/development-standards#prefer-optional-instead-of-null

Reply:
Done. Also clearly delineated the two methods while we're at it.